completions.mjs

```diff
@@ -26,21 +26,16 @@ const openai = new OpenAI({
 });
 
 export async function fetchTextCompletion(tone, prompt, language) {
-    const completion = await openai.chat.completions.create({
-        model: 'gpt-4o-mini',
+    let completion = await openai.chat.completions.create({
+        model: "gpt-4o-audio-preview",
+        modalities: ["text", "audio"],
+        audio: { voice: "alloy", format: "mp3" },
         messages: [
-            { role: 'system', content: `You are a helpful translator. Provide the translation in a phrase. Use example between ### to set the style of response.` },
             {
                 role: 'user',
-                content: `Translate ${prompt} to ${tone} ${language}.
-                ###
-                You can say:
-                ###
-                `,
+                content: `Translate ${prompt} to ${tone} ${language} without repeating the phrase in English.`,
```

Comment:
So the issue here is with prompt injection. This is a pretty simple one, but you generally want to guard against abuse by making the prompt a lot more robust. 

https://platform.lakera.ai/docs/prompt-defense this might be worth a look

Reply:
Appreciate the reminder! I have added a system content message and tried to test a few different scenarios. Please have a look and let me know of other scenarios I may have not considered.